components/__init__.py

```diff
@@ -357,4 +363,15 @@ def define_components(reqs):
                 libs=['protobuf-c'],
                 headers=['protobuf-c/protobuf-c.h'])
 
+    reqs.define('psm2',
+                retriever=GitRepoRetriever(
+                    'https://github.com/intel/opa-psm2.git'),
+                # psm2 hard-codes installing into /usr/...
+                commands=['sed -i -e "s/\\(.{DESTDIR}\\/\\)usr\\//\\1/" ' +
+                          '       -e "s/\\(INSTALL_LIB_TARG=\\/usr\\/lib\\)64/\\1/" ' +
+                          '       -e "s/\\(INSTALL_LIB_TARG=\\)\\/usr/\\1/" ' +
+                          'Makefile compat/Makefile',
+                          'make $JOBS_OPT', 'make DESTDIR=$PSM2_PREFIX install'],
```

Comment:
(pylint-line-too-long) Line too long (81/80)

Reply:
this one seems simple enough...just put the second part on next line?